R/gnps.R

```diff
@@ -0,0 +1,167 @@
+#' @title GNPS spectrum similarity scores
+#'
+#' @description
+#'
+#' The similarity score used in [GNPS](https://gnps.ucsd.edu/) considers, in
```

Comment:
This sentence could be rephrase, as it is isn't entirely clear

> #' The similarity score used in [GNPS](https://gnps.ucsd.edu/) considers, in
> #' addition to directly matching peaks between two spectra, also matches of
> #' peaks with differences in their m/z values that correspond to the difference
> #' of the spectras' precursor m/z. 

What I understand is that peak in spectra x and y are matched twice: (1) using a default approach such as ppm or tolerance and (2) using a fixed delta m/z that is defined by the difference in the two spectras precuror m/z values. If this is the correct interpretations, I would suggest something along the lines of

> The similarity score used in [GNPS](https://gnps.ucsd.edu/) matching peaks between two spectra directly using a user-defined ppm or tolerance as well as a fixed delta m/z that is defined by the difference in the two spectras precuror m/z values.

Reply:
The description below seems much better to me, and points to possible inaccuracies in my understanding. So please, amend as you see fit.

Reply:
Thanks. Yes that's indeed easier.